staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -95,6 +106,13 @@ var baseOpts = []VersionedOptions{
 			ext.Strings(ext.StringsVersion(2)),
 		},
 	},
+	// add set ext library in 1.29
```

Comment:
```suggestion
	// Set library,
```

So we don't have to keep this comment up-to-date if we have to version this library later.